eclipse-ee4j/jaxrs-api
# TCK Migration: Move pending tests from jakartaee-tck/src/com/sun/ts/tests/jaxrs/ee/resource

This PR migrates all tests from [`jakartaee-tck/src/com/sun/ts/tests/jaxrs/servlet3`](https://github.com/eclipse-ee4j/jakartaee-tck/tree/master/src/com/sun/ts/tests/jaxrs/ee/resource).

Linked issues: #1015 

To test the changes :

```
export JAVA_HOME=<JDK11_HOME>
export M2_HOME=<MAVEN_HOME>
export PATH=$M2_HOME/bin:$JAVA_HOME/bin:$PATH
```

In `jaxrs-tck/`:
`mvn clean install`

In `jersey-tck/`:
run migrated tests : `mvn clean verify -Parq-glassfish-managed -Dit.test=jakarta.ws.rs.tck.ee.resource.**`

All tests should succeed.
